src/main/java/org/balanpy/ConfiguracionMascota.java

```diff
@@ -0,0 +1,92 @@
+package org.balanpy;
+
+import java.net.URL;
+import java.time.LocalDate;
+import java.util.ResourceBundle;
+
+import javafx.event.ActionEvent;
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.control.Button;
+import javafx.scene.control.ChoiceBox;
+import javafx.scene.control.DatePicker;
+import javafx.scene.control.ListView;
+import javafx.scene.control.TextField;
+import javafx.scene.image.ImageView;
+import javafx.scene.input.MouseEvent;
+
+public class ConfiguracionMascota implements Initializable{
+	
+	@FXML
+	TextField tf_nombre;
+	@FXML
+	TextField tf_color;
+	@FXML
+	TextField tf_raza;
+	@FXML
+	TextField tf_chip;
+	
+	@FXML
+	ChoiceBox<String> cb_sexomascota;
+	String[] sexo = {"Macho", "Hembra"};
+	@FXML
+	ChoiceBox<String> cb_esterilizada;
+	String[] esterilizada = {"Si", "No"};
+	@FXML
+	ChoiceBox<String> cb_sanguineo;
+	String[] sanguineo = {"DEA-1.1.", "DEA-1.2.", "DEA-3", "DEA-4", "DEA-5", "DEA-6", "DEA-7", "DEA-8"};
```

Comment:
Estos 3 arrays de strings tienen pinta de ser constantes y privados, podríamos marcarlos como tal? Haciéndolos estáticos también nos aseguramos que no se estén creando y borrando cada vez que entramos y salimos de la pantalla.
```suggestion
	static final String[] SEXO = { "Macho", "Hembra" };
	@FXML
	ChoiceBox<String> cb_esterilizada;
	static final String[] ESTERILIZADA = { "Si", "No" };
	@FXML
	ChoiceBox<String> cb_sanguineo;
	static final String[] SANGUINEO = { "DEA-1.1.", "DEA-1.2.", "DEA-3", "DEA-4", "DEA-5", "DEA-6", "DEA-7", "DEA-8" };
```